io/include/pcl/io/image_ir.h

```diff
@@ -56,7 +56,7 @@ namespace pcl
     {
       public:
         using Ptr = shared_ptr<IRImage>;
-        using ConstPtr = shared_ptr<const IRImage>;
+        using ConstPtr = std::shared_ptr<const IRImage>;
```

Comment:
Why are you adding `std` here?

Reply:
A failure of search and replace, apparently.